### ISSUE TITLE
Curl missing in 0.7.3

### ISSUE THREAD

**Issue by nadar** [vercel-community/php#573]
# Bug report

I just deployed with latest vercel-php@0.7.3 and curl extension is missing:

![image](https://github.com/user-attachments/assets/6069be55-8341-427b-976a-c9efbd422699)

![image](https://github.com/user-attachments/assets/7c816ed5-6fa5-4329-8b3f-45af4d5dae95)

but if i take a look at your example php 8.3 app, its available: https://example-php-8-3.vercel.app/

![image](https://github.com/user-attachments/assets/250d819f-12bd-4bf0-a95d-63a854eade17)

- Version: 0.7.3
- No
- No

## Description

This is my vercel.json

```
{
    "version": 2,
    "regions": [
        "fra1"
    ],
    "functions": {
        "api/index.php": {
            "runtime": "vercel-php@0.7.3"
        }
    },
    "routes": [
        {
            "src": "/build/(.*)",
            "dest": "/build/$1"
        },
        {
            "src": "/(.*\\.(?:css|js|png|jpg|jpeg|gif|svg|ico|ttf|woff|woff2|eot|otf|webp|avif|txt))$",
            "dest": "/public/$1"
        },
        {
            "src": "/(.*)",
            "dest": "/api/index.php"
        }
    ],
    "outputDirectory": "public"
}
```

**Comment by nadar** [comment]
+ I changed the `vercel-php@0.7.3` to `vercel-php@0.7.2` just for testing, and its also missing curl. 
+ downgrad to `vercel-php@0.6.2` won't deploy the project at, build error `php: error while loading shared libraries: libssl.so.10: cannot open shared object file: No such file or directory`

**Comment by f3l1x** [comment]
Hi, thank you for report, I will take a look ASAP.

**Comment by nadar** [comment]
let me know if there is anything i can support you in this, the project is super intresting! thank you

**Comment by nadar** [comment]
@f3l1x , if you point me in the right direction, I will send a PR. We would like to push a website into production, but we need `curl`, so we are strongly looking for a fix and are absolutely willing to help here.

**Comment by f3l1x** [comment]
Hi @nadar, I've just test it and seems everythink is OK.

Setup for PHP 8.3 example is this:

URL: https://example-php-8-3.vercel.app/

```
{
  "functions": {
    "api/index.php": {
      "runtime": "vercel-php@0.7.3"
    }
  },
  "routes": [
    { "src": "/(.*)", "dest": "/api/index.php" }
  ]
}
```

<img width="939" alt="image" src="https://github.com/user-attachments/assets/1c17bb44-2d85-4626-a852-17a938c75b4f">

<img width="994" alt="image" src="https://github.com/user-attachments/assets/8890d6ee-0a0e-4140-881c-46f7522d0ce1">

**Comment by nadar** [comment]
Yes, indeed, I just tested it, and I can confirm this. Using Node 18, it works! 😳

Perhaps there's a solution to define this in the vercel.json to help others avoid this trap?

```
{
  "build": {
    "env": {
      "NODE_VERSION": "18.x"
    }
  }
}
```

something like this?

**Comment by Nerexbcd** [comment]
Thank You @nadar, changing to Node 18.X solved my problem
